internal/server/instance/drivers/driver_qemu_templates.go

```diff
@@ -489,6 +490,8 @@ func qemuCPU(opts *qemuCPUOpts, pinning bool) []cfgSection {
 		max := 64
 		if int(cpu.Total) < max {
 			max = int(cpu.Total)
+		} else if opts.cpuRequested > max {
+			max = opts.cpuRequested
```

Comment:
What if cpuCount is greater than cpuRequested? Do you just want Max to be the max of those three values?

Reply:
cpuCount can't be greated than cpuRequested. At the point where it's set, it's either set to 1 (hotplug case) or set to match cpuRequested.

Reply:
cpuRequested is only set in the hotplug case, in all other cases it's just left unset (0) since it's not otherwise useful.